app/src/main/java/model/place/Category.kt

```diff
@@ -56,8 +50,11 @@ class Category() {
         /**
          * Converts a Firestore [document] into a [Category] (null if error during the parsing)
          */
-        fun fromDocument(document: DocumentSnapshot): Category? = document.toObject(Category::class.java)?.apply {
-            id = document.id.toInt()
+        fun fromDocument(document: DocumentSnapshot): Category? {
+            val id = document.id.toInt()
+            val en = document["en"] as? String ?: ""
```

Comment:
Do you want to return a category with default values when a field is not found? Would you rather not return null and optionally log a warning?